resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -600,7 +595,7 @@ The above should yield the prompt to change to what is shown below
 (the exact minor version does not matter as the major version is 14):
 
 ```
-psql (14.4 (Ubuntu 14.4-0ubuntu0.22.04.1))
+psql (14.9 (Ubuntu 14.9-0ubuntu0.22.04.1))
```

Comment:
@joelostblom @chendaniely - don't we want to move to the newest version of postgreSQL, which is 15.4?

Reply:
The reason we explicitly use 14 is that the default package used to be 13, but it seems to have been updated to 14 now. There is no 15 package in the repo, and I vote to keep this simple and use the latest version in the repo unless there are changes to the database course this year that will explicitly require 15. Gittu would know that but I can't find his handle to tag him here so I will ping him on slack

Reply:
OK, once we decide we will have to change either the mac & Windows or the Ubuntu so that they all either install 14 or 15...